Lib/test/test_zoneinfo/test_zoneinfo.py

```diff
@@ -252,6 +252,8 @@ def test_bad_zones(self):
         bad_zones = [
             b"",  # Empty file
             b"AAAA3" + b" " * 15,  # Bad magic
+            # Truncated V2 file (should loop indefinitely)
```

Comment:
```suggestion
            # Truncated V2 file (should not loop indefinitely)
```

I'm so sorry, I forgot a word :cry: